syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -13,6 +13,25 @@
 from syft.exceptions import PureTorchTensorFoundError
 
 
+def _get_maximum_precision():
+    """This function returns the maximum value allowed for precision fractions before the chain decides to use LPT.
+
+    This function can be overridden if the setup requires the use of LargePrecisionTensor from a smaller precision.
+
+    The default value is the size of torch.long
+
+    Returns:
+        The maximum value for precision allowed in this setup
+    """
+    return default_pytorch_maximum_precision()
+
+
+def default_pytorch_maximum_precision():
+    """Dealing with integers > 2**62-1 is not fun with precision tensors.
+    """
```

Comment:
You need to be more precise:
I would suggest doing a check as follows:
if log2(int(value)) + log2(base**precision_fractional) > 62
then go_to_large_precision

But because value can be < 1 or <0
if log2(int(abs(value))+1) + log2(base**precision_fractional) > 62
then go_to_large_precision